Magento.gitignore

```diff
@@ -45,8 +45,43 @@ app/etc/local.xml
 app/.htaccess
 app/locale/
 app/Mage.php
+app/bootstrap.php
 /cron.php
 cron.sh
+/dev/tests/functional/etc
+/dev/tests/functional/lib/Magento/Mtf
+/dev/tests/functional/tests/app/Mage
+/dev/tests/functional/testsuites/Magento/Mtf/TestSuite/InjectableTests/3rdParty.xml
```

Comment:
I gather these are generated tests - could these be simplified into a more wildcard-friendly rule? Or are user tests likely to be put in the same locations?